plugins/forklift/README.md

```diff
@@ -4,10 +4,12 @@ Plugin for ForkLift, an FTP application for OS X.
 
 ### Requirements
 
-* [ForkLift](http://forkliftapp.com/forklift/)
+* [ForkLift](http://www.binarynights.com/forklift/)
 
 ### Usage
 
-* If `fl` is called without arguments then the current folder is opened in ForkLift. Is equivalent to `fl .`
+<code>fl *file*</code>
```

Comment:
I've switched it to `file_or_folder` and added some additional explanatory text.

The formatting doesn't work with regular backticks: then the nested '*...*' text gets rendered as literal asterisks, instead of italicizing it.